src/coreclr/src/jit/rangecheck.h

```diff
@@ -438,7 +438,7 @@ class RangeCheck
     // assumes that the lower range is resolved and upper range is symbolic as in an
     // increasing loop.
     // TODO-CQ: This is not general enough.
-    bool BetweenBounds(Range& range, int lower, GenTree* upper);
+    bool BetweenBounds(Range& range, int lower, GenTree* upper, int arrSize);
```

Comment:
Could you please update the comments for this function? Probably delete this one and add a standard header to the definition with an explanation about the new param.